resources/views/pages-legacy/developerstats.blade.php

```diff
@@ -8,16 +8,35 @@
 $type = requestInputSanitized('t', 0, 'integer');
 $defaultFilter = 7; // set all 3 status' to enabled
 $devFilter = requestInputSanitized('f', 7, 'integer');
-$resolvedForOthersTicketFilter = TicketFilters::AllFilters & ~TicketFilters::StateOpen
-                                            & ~TicketFilters::StateRequest
-                                            & ~TicketFilters::StateClosed;
+$offset = requestInputSanitized('o', 0, 'integer');
+$resolvedForOthersTicketFilter = (
+    TicketFilters::AllFilters
+    & ~TicketFilters::StateOpen
+    & ~TicketFilters::StateRequest
+    & ~TicketFilters::StateClosed
+);
+
+$devStatsList = GetDeveloperStatsFull(100, $offset, $type, $devFilter);
```

Comment:
Suggest lowering the per-page count to 50 or maybe even 25. With pagination, scrolling should be limited.

Reply:
Lowered to 50 - happy to go further down to 25 if you feel this is still excessive.

Reply:
I think lower is probably better. Maybe it's just because the only way to see what page your on is to scroll all the way down.

Reply:
Not a problem at all. Lowered to 25 in latest.